src/syntax/Parser.js

```diff
@@ -2170,6 +2171,13 @@ export class Parser {
         annotations, body);
   }
 
+  parsePropertyVariableDeclaration_(start, isStatic, name, annotations) {
+    var typeAnnotation = this.parseTypeAnnotationOpt_();
+    this.eat_(SEMI_COLON);
```

Comment:
I think the comment is on `this.eat_(SEMI_COLON);` / ";" 
No question typeAnnotation is optional.